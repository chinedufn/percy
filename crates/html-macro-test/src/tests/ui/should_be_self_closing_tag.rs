extern crate virtual_dom_rs;
use virtual_dom_rs::prelude::*;

// We are using open and close tags for a tag that should
// actually be a self closing tag
fn main() {
    html! {
        <br></br>
    };
}
