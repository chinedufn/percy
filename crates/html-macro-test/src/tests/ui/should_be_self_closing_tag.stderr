error: br is a self closing tag. Try "<br>" or "<br />"
 --> $DIR/should_be_self_closing_tag.rs:8:15
  |
8 |         <br></br>
  |               ^^
